README.md

```diff
@@ -35,6 +36,10 @@ Organizations bringing diversity to tech.
 - [Women Techmakers](https://www.womentechmakers.com/) - Google's program that provides visibility, community, and resources for women in technology.
 - [YesWeTech](http://yeswetech.org/) - Feminist community working for an egalitarian and proud culture of women interested in technology.
 
+## Conferences
+The following is a list of conferences that make a special effort to be inclusive and encourage diversity
+
+- [Elixir With Love] (https://www.elixir-with-love.com/) 
```

Comment:
There’s a whitespace here that it’s breaking the markdown formating, maybe we have to fix it before merge the PR :smile: